Contributing-code-to-Oppia.md

```diff
@@ -37,17 +37,19 @@ If you run into any problems along the way, we're here to help! Check out our [[
 
    * (Optional) Consider setting up [automatic auth](https://help.github.com/articles/caching-your-github-password-in-git/) so you don't have to type in a username and password each time you push a change. Note that this isn't an issue if you use SSH.
 
-5. On your browser, consider bookmarking both this wiki page (for easy reference later) and the [Gitter tab](https://gitter.im/oppia/oppia-chat) (so that you can keep abreast of new activity).
+5. On your browser, consider bookmarking the [[guide to making pull requests|Make-a-pull-request]] for easy reference later, as well as the ["my issues" page](https://github.com/issues/assigned) (so that you can keep track of the issues assigned to you).
 
-6. Familiarize yourself with the resources linked to from the sidebar of this page, especially the [[overview of the codebase|Overview-of-the-Oppia-codebase]] the [[coding style guide|Coding-style-guide]], and the [[Frequently Asked Questions|Frequently-Asked-Questions]]. You don't have to read all the other stuff right now, but it's a good idea to be aware of what's available, so that you can refer to it later if needed.
+6. Familiarize yourself with the resources linked to from the sidebar of this page, especially the [[overview of the codebase|Overview-of-the-Oppia-codebase]], the [[coding style guide|Coding-style-guide]], and the [[Frequently Asked Questions|Frequently-Asked-Questions]]. You don't have to read all the other stuff right now, but it's a good idea to be aware of what's available, so that you can refer to it later if needed.
 
-7. Say hi on the [gitter](https://gitter.im/oppia/oppia-chat) chat channel!
+7. Say hi and introduce yourself on [GitHub Discussions](https://github.com/oppia/oppia/discussions/16715)!
 
-8. Join our [gaming event server on discord](https://discord.com/invite/dCJwEAgk3D) for fun activities!
+8. Take up your first starter project! You can find more details below. (Make sure to read and follow the [[PR instructions|Make-a-pull-request]] closely so that your PR review proceeds smoothly.)
 
-9. Take up your first starter project! You can find more details below. (Make sure to read and follow the [[PR instructions|Make-a-pull-request]] closely so that your PR review proceeds smoothly.)
+    * Facing any issues? Please feel free to create a [GitHub Discussion](https://github.com/oppia/oppia/discussions) and get help from the Oppia community.
 
-10. Facing any issues? Please feel free to create a [GitHub Discussion](https://github.com/oppia/oppia/discussions) and get help from the Oppia community.
+    * We also have onboarding mentors who would be more than happy to help you take your first steps in open source. If you like, you can apply for a mentor [here](https://forms.gle/udsRP4WQgLcez9Zm8).
```

Comment:
conditions & information on when mentor is required.

condition: Oppia installation, you choose the issue on which you are working, and you determined why github discussion is ineffective in this situation.

Actually, we also want to gather information on contributor experiences in order to improve github discussion in the future and is need is necessary.

It is must step, otherwise everyone keep asking for 1:1 mentor, and make github discussion ineffective side by side.

Reply:
Do you think we can add that info to the form? That would help over-burdening this page. You might want to take a look at the form if you haven't already...

Reply:
Done.

Reply:
Thanks! Updated the form.